vizro-core/src/vizro/static/css/aggrid.css

```diff
@@ -0,0 +1,46 @@
+#page-container .ag-theme-alpine.vizro {
+  --ag-background-color: var(--main-container-bg-color);
+  --ag-foreground-color: var(--text-primary);
+  --ag-odd-row-background-color: var(--main-container-bg-color);
+  --ag-header-foreground-color: var(--text-primary);
+  --ag-header-background-color: var(--main-container-bg-color);
+  --ag-header-column-resize-handle-display: none;
+  --ag-font-size: 14px;
+  --ag-borders: none;
+  --ag-icon-font-family: agGridBalham;
+  --ag-icon-size: 14px;
+}
+
+#page-container .ag-theme-alpine .ag-header-cell {
+  border-bottom: 1px solid var(--border-subtle-alpha-02);
+  padding-left: 0px;
+}
+#page-container .ag-theme-alpine .ag-header-cell:hover {
+  border-bottom: 1px solid var(--border-hover);
+}
+#page-container .ag-theme-alpine .ag-header-cell:focus {
+  border-bottom: 1px solid var(--border-selected);
+  color: var(--text-primary);
+}
+#page-container .ag-theme-alpine .ag-header-row {
+  color: var(--text-secondary);
+  font-weight: normal;
+}
+
+#page-container .ag-theme-alpine .ag-row {
+  border-bottom: 1px solid var(--border-subtle-alpha-02);
+}
+#page-container .ag-theme-alpine .ag-cell {
+  display: flex;
+  align-items: center;
+  padding-left: 0px;
```

Comment:
This makes it so that you can't align the column in the column defs

Reply:
You should be able to align the columns using the column `type` - see more information and examples in the [dash docs](https://dash.plotly.com/dash-ag-grid/column-definitions#right-aligned-and-numeric-columns)

Try changing the demo `app_dev.py` so the `defaultColDef` includes ` "type": "rightAligned"` like this:
```python
              figure=dash_ag_grid(
                    data_frame=df,
                    className="ag-theme-alpine vizro",
                    # className="ag-theme-custom-theme",
                    defaultColDef={"resizable": True, "sortable": True, "type": "rightAligned",},
                    columnSize="sizeToFit",
                ),
```

Only the headers are right aligned.  The Vizro CSS does't allow the columns to be right aligned
 
![image](https://github.com/mckinsey/vizro/assets/72614349/6165e870-004b-45ab-8c56-97fcc9403614)

Reply:
Ah intersting! I guess we need to do that based on the content type though 🤔

Reply:
Could you elaborate on what the Vizro CSS is going? Maybe we could change that. As it stands it looks a bit confusing :)